files/usr/bin/cinnamon-xlet-makepot

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/python3
+
+import os
+import json
+import subprocess
+import collections
+import argparse
+
+LOCALE_DIR = os.path.join(os.path.expanduser('~'), '.local/share/locale')
+
+USAGE_DESCRIPTION = """\
+Parses files in an applet, desklet or extension's directories, \
+extracts translatable strings, and then generates a ,pot file for them.
+"""
+
+USAGE_EPILOG = """\
+For example:
+cinnamon-xlet-makepot example@mydomain.org/
+
+Will generate a file called example@mydomain.org.pot, in the directory \
+example@mydomain.org/po/ which contains all of the strings that need to be \
+translated. Translators can then use this pot file to generate .po \
+files which contain translations for the extracted strings.
+"""
+
+try:
+    import polib
+except:
+    print("""\
+
+Module "polib" not available.
+
+You will need to install this module using your disrtibution's package manager
+(in debian-based syatems "apt-get install python3-polib")
```

Comment:
Typos:

- **disrtibution's** to **distribution's**
- **syatems to** **systems**.

Reply:
Apparently I don't have spellcheck enabled in sublime on my laptop...
Usually I use my desktop, but I'm having hard drive issues.